src/foremast/awslambda/awslambda.py

```diff
@@ -21,7 +21,8 @@
 from tryagain import retries
 
 from ..exceptions import RequiredKeyNotFound
-from ..utils import get_details, get_properties, get_role_arn, get_security_group_id, get_subnets
+from ..utils import (get_details, get_lambda_arn, get_properties, get_role_arn,
```

Comment:
I would let `isort` manage the import line formatting for consistency. Eventually it'll wrap with parenthesis when it gets too long.